blackcellmagic.py

```diff
@@ -37,10 +37,21 @@ def black(self, line, cell):
         args = magic_arguments.parse_argstring(self.black, line)
         line_length = args.line_length
         if cell:
-            formated = format_str(src_contents=cell, line_length=line_length)
-            if formated and formated[-1] == "\n":
+            # Comment magics
```

Comment:
Shall we leave a detailed comment on:
1. Pattern the regex replacing and what it is replacing it with.
2. Give an example.